### ISSUE TITLE
[Feature]: Official support for anime

### ISSUE THREAD

**Issue by mrjvs** [movie-web/movie-web#346]
### What feature do you want to add?

official support for anime, this means the following:
 - sub and dub choices
 - providers specifically for anime

### Why do you want to have this feature?

A lot of people want to watch anime.

**Comment by arash28134** [comment]
True, the website lacks support for anime. Not an anime fan but I know that many are.

**Comment by atomiciscool** [comment]
w

**Comment by qtchaos** [comment]
One option for getting proper metadata for anime is the [Anify API](https://docs.anify.tv/), though not sure if we should selfhost this or not, since the traffic of movie-web might be too much for it, will have to see about that. [They support searching titles](https://docs.anify.tv/routing/searching) and from a surface level glance, the [info route](https://docs.anify.tv/routing/info) contains everything necessary for our use case.

**Comment by sandeepsandy62** [comment]
I would like to work on it !

**Comment by JipFr** [comment]
A feature this size will likely be done by the core team. Once the structure is up and running, though, we'll likely allow third parties to PR their own anime scrapers — but that's not for a little bit.

**Comment by TheScreechingBagel** [comment]
> One option for getting proper metadata for anime is the [Anify API](https://docs.anify.tv/)

another option: anilist seems to have a free api (for "non-commercial use") https://github.com/AniList/ApiV2-GraphQL-Docs
